README.md

```diff
@@ -23,16 +23,20 @@ this environment here: https://hub.docker.com/r/2performantirina/middleman-and-i
 
 ## Deploying
 
-Change the directory to `build`, and commit the changes.
+Run `bundle exec middleman build`, which will compile the site to `docs`.
+
+To deploy to staging you can create a PR against https://github.com/RSpec-Staging/rspec-staging.github.io/
+or you can push temporarily with:
 
-To publish to the staging site (http://rspec-staging.github.io/), run:
 ```
-git push staging master
+git push staging <branch>:source --force
 ```
 
-To publish to the main documentation site (http://rspec.info/), run:
+To deploy to production you should create a PR against https://github.com/rspec/rspec.github.io/
+or run:
+
 ```
-git push origin master
+git push origin source
```

Comment:
Hm. I think this needs to be changed.
So the workflow is:
1. Make your changes to `source` directory
2. Run `middleman build`
3. `git push staging`
4. Check if everything is in order on https://rspec-staging.github.io
5. Merge the pull request (and this automatically "pushes" to `origin`'s `source` branch, no manual push needed.

Do you agree?

Reply:
I'm happy to reword the entire section, but this is about deploying, step 1 is irrelevant, thats assumed to be done, step 2 is mentioned above, step 3 is above, and step 4 is also in the sentance above.

Reply:
I mean we could stop pushes to source entirely and remove this line :joy:

Reply:
Right. My whole point is the step 5.
So instead of pushing to `origin`'s `source`, to deploy to http://rspec.info, one need to merge the pull request.

This way you may protect the `source` branch even from accidental pushes from us, admins.
No potential to skip a change if e.g. two pull requests are open, and:
1. pull request A is pushed to `origin/source`
2. pull request A is merged (no changes, everything is pushed already)
3. pull request B is pushed to `origin/source` (and fail without `--force`)
3.a pull request B is push-forced in a rush to `origin/source`
4. pull request B is merged. changes from pull request A are lost with no trace

I would prefer to avoid such a risk.

Reply:
Its not a step 5, its step 4a step 4b

Reply:
If a PR is merged pushing has no effect

Reply:
> I mean we could stop pushes to source entirely and remove this line

👍 yep!

Reply:
At least to `origin`.

It seems to be very convenient to push to `rspec-staging/source`, much better than opening a second pull request just to check the changes. And keeping those two PRs in sync.

Reply:
> If a PR is merged pushing has no effect

Right. Just afraid if someone will push-force to `origin/source`. That would remove the merge commit for pull request A just as well.